src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java

```diff
@@ -122,17 +123,16 @@ public void updateEserviceFrequency(UpdateEserviceFrequencyDto inputData)
         .pollingStartTime(inputData.getNewPollingStartTime())
         .pollingEndTime(inputData.getNewPollingEndTime());
     eserviceRepository.save(eServiceToUpdate);
-
-    log.info("Eservice " + eServiceToUpdate.eserviceId() + " with version "
-        + eServiceToUpdate.versionId() + " has been updated with startTime: "
-        + eServiceToUpdate.pollingStartTime() + " and endTime: " + eServiceToUpdate.pollingEndTime()
-        + " and frequency: " + eServiceToUpdate.pollingFrequency());
+    logger.logMessageEservicePollingConfigUpdated(eServiceToUpdate.eserviceId(), eServiceToUpdate.versionId(),
```

Comment:
Maybe it is better to pass the whole object

Reply:
Done and pushed